moduleroot/.gitlab-ci.yml.erb

```diff
@@ -91,6 +91,22 @@ before_script:
   allow_failure: true
 <%         end -%>
 <%       end -%>
+<%       if options['except'] -%>
+<%         if options['except'].has_key?(check) -%>
+  except:
+<%           options['except'][check].each do |ref| -%>
```

Comment:
This might need to guard against `options['except'][check]` being `nil` since `has_key?` won't protect from the value being `nil`.

Reply:
```suggestion
<%           options['except'][check] && options['except'][check].each do |ref| -%>
```